src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import { CheckIcon, CloseIcon } from '../../assets/svgs'
+import styles from './TodoItem.module.scss'
+import { classNames } from '../../styles'
+
+const cx = classNames.bind(styles)
+
+function TodoItem({ todo, setTodoList }) {
+  const handleChange = (e) => {
+    const { dataset, checked } = e.currentTarget
+    const { id } = dataset
+
+    setTodoList((prev) => {
+      const targetIndex = prev.findIndex((todoItem) => todoItem.id === Number(id))
+      const newList = [...prev]
+      newList[targetIndex].done = checked
+
+      return newList
+    })
+  }
+  const handleDelete = (e) => {
+    const { id } = e.currentTarget.dataset
+
+    setTodoList((todoLists) => todoLists.filter((listItem) => listItem.id !== Number(id)))
```

Comment:
```suggestion
    setTodoList((prev) => prev.filter((listItem) => listItem.id !== Number(id)))
```
위(L13)에서 `prev`를 써서 통일감있게 여기서 `prev`로 가면 좋을 것 같아요!